bcipy/acquisition/devices.json

```diff
@@ -49,6 +52,7 @@
         "channels": [ "P4", "Fz", "Pz", "F7", "PO8", "PO7", "Oz", "TRG" ],
         "sample_rate": 300.0,
         "connection_methods": ["TCP", "LSL"],
-        "description": "Alias to DSI-VR300 for backwards compatibility."
+        "description": "Alias to DSI-VR300 for backwards compatibility.",
```

Comment:
If we don't use them in practice anymore (the backward compatibility devices), let's remove them for clarity! I don't think any other users rely on it being here.

Reply:
They are still used in demo code and servers. I'll try to phase them out, though.